ccloud/docs/beginner-cloud.rst

```diff
@@ -1,46 +1,43 @@
-
-.. _beginner-cloud:
+.. _ccloud-cli-tutorial:
 
 Tutorial: |ccloud| CLI
 =======================
 
 Overview
 --------
 
-This tutorial will show you how to use the `Confluent Cloud CLI
+This tutorial shows you how to use the `Confluent Cloud CLI
 <https://docs.confluent.io/current/cloud/cli/install.html>`__ to interact with
 your `Confluent Cloud <https://confluent.cloud/login>`__ cluster.
+Following the workflow in this tutorial, you complete the following steps:
```

Comment:
What do you think about something like this: "Following the workflow in this tutorial, you accomplish the following steps:" ?

Reply:
Ack & done.